examples/jsonnetfile.lock.json

```diff
@@ -1,14 +1,15 @@
 {
-    "dependencies": [
-        {
-            "name": "grafonnet",
-            "source": {
-                "git": {
-                    "remote": "https://github.com/grafana/grafonnet-lib",
-                    "subdir": "grafonnet"
-                }
-            },
-            "version": "bcd95ffa00fc4a58d34832f88f4b366effeb63ad"
+  "dependencies": [
```

Comment:
not sure why this file changed?

Reply:
This file was still a jsonnet-bundler v0.1.0 generated one. It's fine to have this updated to v0.2.0 :) Next version of jsonnet-bundler will have this properly versioned :upside_down_face: